GVRf/Framework/framework/src/main/jni/engine/renderer/renderer.cpp

```diff
@@ -755,6 +754,11 @@ void Renderer::renderMaterialShader(RenderState& rstate, RenderData* render_data
                  glLineWidth(1.0f);
              }
          }
+         if (shader->getProgramId() != -1)
+         {
+             mesh->bindVertexAttributes(shader->getProgramId());
```

Comment:
if we are retrieving attrib locations from the program object, we dont need mesh->bindVertexAttributes() call, it just creates attribute locations, and fill the map of attribute keys which we use for generating VAO but in our case we don't need attribkeys map in mesh. we can remove this call.